graphics/inkscape-gtk3-devel/Portfile

```diff
@@ -73,23 +74,18 @@ depends_lib-append \
                     port:py${python_version}-lxml \
                     port:py${python_version}-numpy
 
-post-patch {
-    reinplace "s|\"python-interpreter\", \"python\"|\"python-interpreter\", \"python${python_major}.${python_minor}\"|g" ${worksrcpath}/src/extension/implementation/script.cpp
-    reinplace "s|^#include \"Object.h\"|#include \"${prefix}/include/poppler/Object.h\"|" ${worksrcpath}/src/extension/internal/pdfinput/pdf-parser.h
-    reinplace "s|^#include \"Object.h\"|#include \"${prefix}/include/poppler/Object.h\"|" ${worksrcpath}/src/extension/internal/pdfinput/pdf-parser.cpp
-    reinplace "s|lib/inkscape|lib|" ${worksrcpath}/src/CMakeLists.txt
-}
+patchfiles         patch_lib_dir.patch
+patch.pre_args     -p1
 
 # py-numpy is currently not universal (#48263).
-
 universal_variant no
 
 # clang-425.0.28 cannot handle glibmm's headers
 # allow build with more modern gcc on 10.5 and earlier
 compiler.blacklist-append {clang < 500} *gcc-3.* *gcc-4.*
 
 configure.args-append \
-    -DWITH_DBUS:BOOL=ON \
+    -DWITH_DBUS:BOOL=OFF \
```

Comment:
This change seems significant. Did you intend to do this? Does the port need to still depend on `dbus-glib` if so?

Reply:
It was giving me errors related to linking to the `Dbus` session, even if the session was active (but the app was running ok). I also [sent a mail on the mailing list](https://lists.macports.org/pipermail/macports-dev/2019-July/040940.html). So I disabled it. 

I guess like this it does not need to depend on `dbus-glib` anymore.